examples/verifyIdToken.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const OAuth2Client = require('../build/src/index').OAuth2Client;
+const http = require('http');
+const url = require('url');
+const querystring = require('querystring');
+const opn = require('opn');
+
+// Download your OAuth2 configuration from the Google
+const keys = require('./oauth2.keys.json');
+
+/**
+ * Start by acquiring a pre-authenticated oAuth2 client.
+ */
+async function main() {
+  try {
+    const oAuth2Client = await getAuthenticatedClient();
+
+    // Verify the id_token, and access the claims.
+    const ticket = await oAuth2Client.verifyIdToken(
+        oAuth2Client.credentials.id_token,
+        keys.web.client_id);
+    console.log(ticket);
+
+    // You can use this info to get user information too.
+    const url = `https://www.googleapis.com/plus/v1/people/me`;
+    const res = await oAuth2Client.request({url})
+    console.log(res.data);
+  } catch (e) {
+    console.error(e);
+  }
+  process.exit();
```

Comment:
I'm not sure why this is needed.

Reply:
There is some problem with the http server and opn modules grabbing handles that prevent the app from finishing naturally.  I could fix it, but it adds another ~50 LOC to the sample that don't add much value.